en/developer_guide/routing.rst

```diff
@@ -102,27 +114,30 @@ Declaring differents loaders as services in ``routing.yml`` is mandatory to gene
         resource: '.'
         type: orchestra_redirection
 
-See `Symfony Loader documentation`_ for more information.
+You also have to modify the project configuration, in the ``config.yml`` file :
+
+.. code-block:: yaml
+
+    open_orchestra_front:
+        routing_type: file
+
+How to choose between both of the solution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this part, I would not tell you which routing type you should use, but only list a some condition to use
```

Comment:
My previous comment disappeared, so I copy/paste it : "Avoid ``I`` or ``we`` in the doc, it's not a formation report :D"